Tests/CloudFoundryConfigTests/config_example.json

```diff
@@ -21,22 +21,43 @@
         ]
       }
     ],
-    "WatsonConversation": [
+    "imfpush": [
+      {
+        "credentials": {
+          "appGuid": "<appGuid>",
+          "url": "http://imfpush.ng.bluemix.net/imfpush/v1",
+          "admin_url": "//mobile.ng.bluemix.net/imfpushdashboard",
+          "appSecret": "<appSecret>",
+          "clientSecret": "<clientSecret>"
+        },
+        "label": "imfpush",
+        "name": "PushNotificationService",
+        "plan": "Lite",
+        "provider": null,
+        "syslog_drain_url": null,
+        "tags": [
+          "mobile",
+          "ibm_created",
+          "ibm_dedicated_public"
+        ]
+      }
+    ],
+    "conversation": [
```

Comment:
Note that I updated the Watson Conversation service name to `conversation` here.